test/parallel/test-internal-fs-syncwritestream.js

```diff
@@ -66,6 +66,30 @@ const filename = tmpdir.resolve('sync-write-stream.txt');
   assert.strictEqual(stream.fd, null);
 }
 
+// Verify that the callback will be called when already destroy()ed.
+{
+  const fd = fs.openSync(filename, 'w');
+  const stream = new SyncWriteStream(fd);
+  const theErr = new Error('my error');
+  const cb = () => {};
+
+  stream.on('close', common.mustCall());
+  assert.strictEqual(stream.destroy(), stream);
+  stream.destroy(theErr, common.mustCall(cb));
```

Comment:
Using `writable.destroy()` with a callback is undocumented and I do not think we should use it here.

Reply:
Thank you for your comment, I agree with your opinion